Add marchitecture

cristiklein opened the pull request:
Added a simplified diagram with all Compliant Kubernetes components.

simonekman reviewed (changes requested):
Looks great but I'm missing some logos :) 

What about backups and Velero?

"Kured" could go in the security box? 

Does TimeScale DB have a logo? If so that should go in the top box.

Perhaps ArgoCD and Jaeger could be in there as well but with a marking showing they are "preview features"?

cristiklein commented:
> Looks great but I'm missing some logos :)

Trying hard not to feel embarrassed that you know Compliant Kubernetes better than me. :smile:

> What about backups and Velero?

Added Velero and Rclone in the "disaster recovery" box.

> "Kured" could go in the security box?

Done! I moved Harbor to additional service, to declutter the security box a bit.

> Does TimeScale DB have a logo? If so that should go in the top box.

Done!

> Perhaps ArgoCD and Jaeger could be in there as well but with a marking showing they are "preview features"?

We can do that at the end of Q4. (They are currently in alpha, so we should not show them.)

PTAL now.

llarsson reviewed (approved):
Nice! LGTM!

simonekman reviewed (approved):
Great, I'm very happy with this version!
